### PR TITLE
Starship improvements

### PR DESCRIPTION
**Pull request by Buckram123**
### Checklist

- [x] Changelog updated.
- [ ] Docs updated.

### CODE REVIEW

File: cw-orch-daemon/src/sync/builder.rs [hunk 1]
```diff
@@ -128,6 +130,13 @@ impl DaemonBuilder {
         self
     }
 
+    /// Load network from `$HOME/.cw-orchestrator/networks.json`
```
Comment (CyberHoward) [inline cw-orch-daemon/src/sync/builder.rs:133]: Do we want to change this to toml before merging?

**Review by CyberHoward — APPROVED** [review]
LGTM! Just want to see if we don't want to use toml instead of json for the network overwrite?

File: cw-orch-daemon/src/sync/builder.rs (hunk 1 above)
Comment (Buckram123) [reply cw-orch-daemon/src/sync/builder.rs]: Changed!

File: cw-orch-daemon/tests/mock_networks.toml
Comment (Buckram123) [inline cw-orch-daemon/tests/mock_networks.toml:1]: Thiis is 514 characters which is still lower than minimized json 531, so only benefits here

**Pull request merged**